index.js

```diff
@@ -0,0 +1,61 @@
+const display = document.getElementById("display");
+const equalBtn = document.getElementById("equal");
+const clearBtn = document.getElementById("clear");
+let num1 = null;
+let num2 = null;
+let mainOperator = null;
```

Comment:
Use array destructuring to initialize the variables num1, num2, and mainOperator to null.
`let [num1, num2, mainOperator] = [null, null, null];`